atat_provisioning_wizard_api.yaml

```diff
@@ -1040,18 +1042,16 @@ components:
   examples:
     PortfolioDraftSummaryEx:
       value:
-        - id: "dc2bbee6-8cdb-477e-a363-f9f1593a0a9b"
-          status: "not_started"
-          created_at: "2021-08-03T16:19:51.686Z"
-          updated_at: "2021-08-03T16:19:51.686Z"
-        - id: "c3e8c444-a971-4515-bc43-3cdfef072ccc"
-          status: "in_progress"
-          created_at: "2021-08-03T16:19:24.362Z"
-          updated_at: "2021-08-03T16:19:24.362Z"
-        - id: "3f44ef42-2322-4a42-a381-6770e18eceba"
-          status: "complete"
-          created_at: "2021-08-03T16:21:07.978Z"
-          updated_at: "2021-08-03T16:21:07.978Z"
+        id: "dc2bbee6-8cdb-477e-a363-f9f1593a0a9b"
+        created_at: "2021-08-03T16:19:51.686Z"
+        updated_at: "2021-08-03T16:19:51.686Z"
+        status: "not_started"
+        name: "Mock Portfolio"
+        description: "Mock portfolio description"
+        num_portfolio_managers: 2
+        num_task_orders: 2
+        num_applications: 2
+        num_environments: 2
```

Comment:
Should this still be a list since `/portfolioDrafts` returns a list/array? Even if it's just a list of one element?

Reply:
Yes, you're correct.  `PortfolioDraftSummaryEx` should be an example of the _array_ of `PortfolioDraftSummary`.

I was thinking about it as an example of a single PortfolioDraftSummary.  That's not correct.
```
              schema:
                type: array
                items:
                  $ref: '#/components/schemas/PortfolioDraftSummary'
              examples:
                PortfolioDraftSummaryEx:
                  $ref: '#/components/examples/PortfolioDraftSummaryEx'
```
![image](https://user-images.githubusercontent.com/77642966/135908509-8009ac51-1654-43b9-8bed-e81b73236721.png)

Reply:
Fixed in 5c131593afe69d7fff0d1296100f65589652cb5b

Reply:
`PortfolioDraftSummaryListEx` or `PortfolioDraftSummaryArrayEx` would be clearer.

Reply:
Renamed in 3b90aa13b2adbf91275632fd21039a490104e06e